packages/schematics/angular/service-worker/index.ts

```diff
@@ -60,7 +60,7 @@ function updateConfigFile(options: ServiceWorkerOptions, root: string): Rule {
 
     const config = getProjectConfiguration(workspace, options);
     config.serviceWorker = true;
-    config.ngswConfigPath = `${root.endsWith('/') ? root : root + '/'}ngsw-config.json`;
+    config.ngswConfigPath = `${root && !root.endsWith('/') ? root + '/' : root}ngsw-config.json`;
```

Comment:
If root happens to be `undefined` the string will be `undefinedngsw-config.json`

Reply:
If `project.root` is ever undefined. It will break half of our workflow :p